css/README.md

```diff
@@ -181,6 +182,8 @@ The `sections/` directory is where styling specific to a page or section should
       color: var(--brand-primary);
     }
     ```
+    
```

Comment:
I forgot this `sections/` "section" was in here. I wonder if it needs to be? I feel like it's not a thing we really do.